### DISCUSSION TITLE
Update & Status on your submissions

### DISCUSSION THREAD

**Discussion by bdougie** [open-sauced-craftwork/browser-extension#55] [Announcements]
First off, I would like to thank you all for your submissions to the OpenSauced Octernship. Some excellent ideas were submitted and some great conversations these past 30 days. 

I appreciate those who submitted early and were patient while we figured out to work with the submissions. I also appreciate the folks who were patient and waiting for their reviews from me and @nickytonline. 

## Selection process.
**We have selected two Octerns**, and they will be getting a notice from the GitHub Education team directly. The process was hard, as there were 500 submissions and the project was intentionally vague to see how the students would approach it with their skillset. 

### **PR description**
Because OpenSauced is an open-sourced company and projects, the browser extension needed individuals with exceptional communication skills. I did not apply much consideration to the code you wrote, but much more reflection on your PR description and interactions in the GitHub Discussions and Discord. 

For those who received comments about adding a PR description, this was your opportunity for a second chance. There were a number of you that submitted context about your submission in the project README, this was ok, but the proper interaction in a code review is to make it as easy as you can for the review to review your code. 

I cover this in a YouTube video I created - https://www.youtube.com/watch?v=cuMeC-eZJJ4

## What's next?
Thank you so much for the time and submission. If you were not selected, I encourage you to keep your open-source journey from ending here. OpenSauced exist to motivate contributions to open-source project. 

Please [consider highlighting](https://www.youtube.com/watch?v=yOwQJ3zGPpI) your future contributions.

We will set up a call for those selected in the next few weeks.

**Comment by diivi** [comment]
Best of luck to all contributors and the OpenSauced team! Looking forward to using the extension when it's live!
And thanks for the feedback and interactions, this was an amazing experience!

**Comment by thekingslee** [comment]
Cool stuff. Good luck to the contributors and thank you @bdougie.

**Comment by shuklaritvik06** [comment]
Those who are not selected will also get some notification from GitHub Education?

**Comment by Pranav-yadav** [comment]
Late to the party... Just finished with the exams.

Thanks for the awesome reviews, feedbacks and interactions @bdougie, @nickytonline and @0-vortex 🙌. It was definitely an awesome experience and got to learn a lot during this short period.

Looking forward to contribute more to Open Source.

**Comment by bdougie** [comment]
Hi everyone, messaging to share we have sent out offers to two students who have both accepted to join the OpenSauced team next week to build out our new chrome extension.

This was a hard decision and I want to thank everyone who participated and took the time to show off their skills. Everyone who submitted had the technical skills, but our focus was on communication. Open source requires great communication to assist in coming to resolution and shipping features. 

All work will happen in a public repo: https://github.com/open-sauced/browser-extensions/

I encourage you to stick around in the community if you were not selected. We have a lot planned for the product and hope we can start assisting others in finding paid and unpaid open-source work to expand resumes. 

Be sure to keep your [OpenSauced](https://opensauced.pizza) profile up to date and stay active in our Discord so you know when the latest
